### DISCUSSION TITLE
New Release: 3.3.1 Beta 1

### DISCUSSION THREAD

**Discussion by Xanashi** [Xanashi/Icaros#45] [Releases]
I forgot to auto-generate a discussion for this release.
Please discuss here. The changelog and download links can be [found here](https://github.com/Xanashi/Icaros/releases/tag/v3.3.1b1).

**Comment by Klaus1189** [comment]
Thank you very much ❤️❤️❤️

**Comment by Tarek-Hasan** [comment]
Thank you for the release.
I have a question. Why .NET Framework 4.6? 4.6 was released back in 2016. Why not newer version 4.8?
Or better support .NET 6 LTS.

**Reply by Xanashi** [reply]
I am considering upgrading to a higher version, however 4.6 is the last version supported on Windows Vista,
and at this point in time Icaros still works on Vista, and has Vista users, so I wish to keep this support going for bit longer.
If there is a way to upgrade further without ending support, I will definitely do it.

**Reply by Tarek-Hasan** [reply]
None should use vista. It's the worst Windows version in terms of performance. It's buggy and slow. Win7 is much better; also it has better driver and software support.
